packages/webapp/src/components/Task/FieldWorkTask/index.jsx

```diff
@@ -13,23 +17,14 @@ const PureFieldWorkTask = ({
   disabled = false,
 }) => {
   const { t } = useTranslation();
+
+  const { fieldWorkTypes = [] } = useSelector(fieldWorkSliceSliceSelector);
```

Comment:
The purpose of component/container folder is separate testable/untestable components.
In component folder, all components do not use selectors or dispatch/network requests so that components can be unit tested without mocking redux store/network requests.